service/history/workflow/workflow_task_state_machine.go

```diff
@@ -1159,8 +1161,8 @@ func (m *workflowTaskStateMachine) convertSpeculativeWorkflowTaskToNormal() erro
 		return serviceerror.NewInternal(fmt.Sprintf("it could be a bug, scheduled event Id: %d for normal workflow task doesn't match the one from speculative workflow task: %d", scheduledEvent.EventId, wt.ScheduledEventID))
 	}
 
-	if wt.StartedEventID != common.EmptyEventID {
-		// If WT is started then started event is written to the history and
+	if wtAlreadyStarted := wt.StartedEventID != common.EmptyEventID; wtAlreadyStarted {
```

Comment:
After we agreed that correct abbreviation for "workflow task" is WFT (not WT), I am slowly changing it on the lines I touch or add.
Like your `var` here. Nice technique.

Reply:
I had renamed it at first, but then the rest of this huge file would still be `wt`. I thought a sweeping change to rename things would be better than doing it piece-meal?.